DOCS/man/options.rst

```diff
@@ -1550,19 +1550,22 @@ Video
     full size, even if parts of the video are not visible due to panscan or
     other options).
 
-    For example, displaying a video fullscreen on a 1920x1080 screen with
-    ``--video-pan-x=-0.1`` would move the video 192 pixels to the left and
-    ``--video-pan-y=-0.1`` would move the video 108 pixels up.
+    For example, displaying a video scaled to 1920x1080 due to video-zoom or
+    other options on a 1280x720 mpv window, with ``--video-pan-x=-0.1`` would
+    move the video 192 pixels to the left and ``--video-pan-y=-0.1`` would move
+    the video 108 pixels up.
 
     This option is disabled if the ``--no-keepaspect`` option is used.
 
 ``--output-pan-x=<value>``, ``--output-pan-y=<value>``
     Same as ``--video-pan-x`` and ``--video-pan-y`` except unit of the value is
     in fractions of the size of the destination rectangle.
 
-    For example, displaying a video fullscreen on a 1920x1080 screen with
-    ``--output-pan-x=-0.1`` would move the video 192 pixels to the left and
-    ``--output-pan-y=-0.1`` would move the video 108 pixels up.
+    For example, displaying a video scaled to 1920x1080 due to video-zoom or
+    other options on a 1280x720 mpv window, with ``--output-pan-x=-0.1`` would
+    move the video 128 pixels to the left and ``--output-pan-y=-0.1`` would move
+    the video 72 pixels up. If the video isn't scaled, most of the time this and
```

Comment:
Is this example right? I thought it would be the same unless you used `--video-zoom` or `--video-scale`. So. e.g. if you only fullscreen the video, it should be the same right?

Reply:
No, you're right. It's ambiguous wording. I'll drop the usage of the word scale and explicitly mention options. Or maybe drop this line altogether...

Reply:
Opted to remove it instead, we generally don't mention other options in examples for options and it was still confusing anyway.

Reply:
It's still there though.

Reply:
Hmm I thought you meant the "If the video isn't scaled, most of the time and ..." line. If you mean the example as a whole, then I think it's quite accurate unless I'm missing something silly here

The values are not the same, and I do explicitly mention "video-zoom or others".

Reply:
Oh I thought you meant remove the whole thing lol. I guess it's alright.